src/styles/block/card.scss

```diff
@@ -0,0 +1,84 @@
+a,
+p,
+h1,
+h2 {
+  margin-bottom: 0;
+}
+
+.card {
+  width: 198px;
+  background-color: $white;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main;
+  }
+  &__box {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+  }
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+  &__text {
+    margin-top: 3px;
+    margin-bottom: 0;
+    line-height: 16px;
+    color: $secondary;
+  }
+  &-reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 15px;
+  }
+  &-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+    &__title {
+      margin-top: 0;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary;
+    }
+    &__number {
+      margin-top: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main;
+    }
+  }
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+    width: 100%;
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    height: 40px;
+    background-color: $blue;
+    border: 1px solid $blue;
+    color: $white;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+  &:hover &__button {
+    background-color: $white;
+    color: $blue;
+  }
```

Comment:
these styles should be applied only when cursor is over the button, not the whole card

Reply:
Thank you so much, I misunderstood, hah.